locale/translations/fr_FR.json

```diff
@@ -152,7 +152,8 @@
     "page.keyboard_shortcuts.open_original_same_window": "Ouvrir le lien original dans l'onglet en cours",
     "page.keyboard_shortcuts.open_comments": "Ouvrir le lien des commentaires",
     "page.keyboard_shortcuts.open_comments_same_window": "Ouvrir le lien des commentaires dans l'onglet en cours",
-    "page.keyboard_shortcuts.toggle_read_status": "Basculer entre lu/non lu",
+    "page.keyboard_shortcuts.toggle_read_status_next": "Basculer entre lu/non lu, se concentrer ensuite",
+    "page.keyboard_shortcuts.toggle_read_status_prev": "Basculer entre lu/non lu, focus précédent",
```

Comment:
I thought Google Translate would have been better. Here are more suitable French translations:

```
Basculer entre lu/non lu, et changer le focus sur l'élément suivant
Basculer entre lu/non lu, et changer le focus sur l'élément précédent
```

Reply:
Updated.

It's hard for google translate because it's just "focus next" / "focus previous" in a single sentence, without additional context. The Portuguese translation was also a bit off.